docs/KnownIssues.md

```diff
@@ -0,0 +1,27 @@
+# Known Issues
+
+This page lists all the ongoing known issues in PowerApps-Language-Tooling for which we have workarounds. This document will be continuously updated as issues and workarounds are discovered.
+
+## Warning PA2001: Checksum mismatch. AppTests\2.json checksum does not match on extract
+
+This is a known issue with apps developed with a previous version of Studio, and has now been resolved. This error is known to occur due to a stray empty test step that gets added when unpacking the msapp. This can be resolved by adding an action to the empty test step and then deleting the test entirely by the following steps -
+
+1. Open the msapp back into the maker portal
```

Comment:
Not sure if "maker portal" is a term we use in client docs. Looking around it seems like we may want to use the term "Power Apps Studio".